snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/ClosedChannel.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.kotlinmpplib
+
+import breez_sdk.*
+
+class ClosedChannel {
+    func prepareSweep(sdk: BlockingBreezServices, feeRate: int) {
+        // ANCHOR: prepare-sweep 
+         val satPerVbyte = feeRate
+         var destinationAddress = "bc1.."
+         try {
+            val req = PrepareSweepReq(satPerVbyte, destinationAddress)
+            prepareSweepResp = sdk.prepareSweep(req)
+         }
+         catch (e: Exception) {
+            // handle error
+        }
+        // ANCHOR_END: prepare-sweep 
+    }
+
+    func sweep(sdk: BlockingBreezServices, satPerVbyte: int, toAddress) {
```

Comment:
For some reason satPerVbyte is uint32 here, so UInt
```suggestion
    func sweep(sdk: BlockingBreezServices, satPerVbyte: UInt, toAddress: String) {
```

Reply:
Added